specification/automation/resource-manager/Microsoft.Automation/stable/2018-01-15/dscNode.json

```diff
@@ -0,0 +1,957 @@
+          },
+          {
+            "name": "nodeId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The node id."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscNode"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      },
+      "patch": {
+        "tags": [
+          "DscNode"
+        ],
+        "operationId": "DscNode_Update",
+        "description": "Update the dsc node.",
+        "externalDocs": {
+          "url": "http://aka.ms/azureautomationsdk/dscnodeoperations"
+        },
+        "x-ms-examples": {
+          "Update a node": {
+            "$ref": "./examples/updateDscNode.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "./definitions.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/AutomationAccountNameParameter"
+          },
+          {
+            "name": "nodeId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Parameters supplied to the update dsc node."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/DscNodeUpdateParameters"
+            },
+            "description": "Parameters supplied to the update dsc node."
+          },
+          {
+            "$ref": "./definitions.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "./definitions.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/DscNode"
+            }
+          },
+          "default": {
+            "description": "Automation error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./definitions.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Automation/automationAccounts/{automationAccountName}/nodes": {
+      "get": {
+        "tags": [
+          "DscNode"
+        ],
+        "operationId": "DscNode_ListByAutomationAccount",
```

Comment:
>DscNode_ListByAutomationAccount [](start = 24, length = 31)

This operation could also use x-ms-odata

Reply:
Same as above: Thanks. I'll talk with the team and take this up as a separate PR as it is not a blocker right now.

Reply:
Done

Reply:
Fixed